core/jazz_ui/protractor.conf.js

```diff
@@ -54,7 +55,7 @@ exports.config = {
       beautify: true,
       indentationLevel: 4 
     }));
-    browser.manage().timeouts().implicitlyWait(600000);
+    browser.manage().timeouts().implicitlyWait(2000000);
```

Comment:
> 
browser.manage().timeouts().implicitlyWait(**2000000**);
    browser.manage().window().setSize(**1600, 1000**);

Can we use params instead of hard coded values?

Reply:
Lets address this later